### PR TITLE
Home page

### PR DESCRIPTION
**Pull request by paultursuru**
`rails g controller Welcome index` pour qu'on ai de quoi bosser la suite

### CODE REVIEW

File: app/controllers/welcome_controller.rb [hunk 1]
```diff
@@ -0,0 +1,4 @@
+class WelcomeController < ApplicationController
+  def index
+  end
```
Comment (svilder) [inline app/controllers/welcome_controller.rb:3]: Not sure a "Welcome" controller is needed. Maybe just a Home action in Pages controller ?

File: app/views/layouts/_nav-bar.html.erb [hunk 2]
```diff
@@ -1,7 +1,8 @@
 <div class="navbar navbar-expand-sm navbar-light navbar-lewagon">
   <%= link_to "#", class: "navbar-brand" do %>
     <%= image_tag "https://raw.githubusercontent.com/lewagon/fullstack-images/master/uikit/logo.png" %>
-    <% end %>
+  <% end %>
+    <p class="main-title">le wagon alumni</p>
 
   <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
     <span class="navbar-toggler-icon"></span>
```
Comment (svilder) [inline app/views/layouts/_nav-bar.html.erb:8]: Navbars are more classified in "Shared" folder. Also, filename should be as simple as possible : _navbar.html.erb (no - )

File: config/routes.rb [hunk 3]
```diff
@@ -1,3 +1,5 @@
 Rails.application.routes.draw do
+  get 'welcome/index'
+  root 'welcome#index'
   # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
 end
```
Comment (svilder) [inline config/routes.rb:5]: Same issue with Welcome instead of Home. To be thought and discussed with the team

**Review by svilder — CHANGES_REQUESTED** [review]

File: app/controllers/welcome_controller.rb (hunk 1 above)
Comment (paultursuru) [reply app/controllers/welcome_controller.rb:3]: Agreed, doing this on a new branch right now ;)

**Comment by paultursuru** [comment]
I'll close this branch when pages is merged

**Pull request closed without merging**